src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {WebexAvatar} from '../';
+import {useMeeting, useVideo} from '../hooks';
+
+import './WebexLocalMedia.scss';
+
+/**
+ * Webex Local Media component displays the user's local video
+ * previous to joining a meeting from a given meetingID.
```

Comment:
I think you should remove this part of the comment. Local media will be used in-meeting as well, for meeting "self-view"